cmd/bbolt/README.md

```diff
@@ -0,0 +1,374 @@
+# Introduction to bbolt command line
```

Comment:
I see we have a mix of `bbolt help` and additional descriptions of command. Could we improve `bbolt help` at the same time? Just add the description you added to bbolt help

Reply:
>  Could we improve bbolt help at the same time? Just add the description you added to bbolt help

I didn't get it exactly, I should a `help` command for every other command
`bbolt <command> -h` eg: `bbolt info -h` , Is this you are saying ?

Reply:
Yes

Reply:
I have mentioned this in `help` command:
```
you can use `help` with any command: `bbolt [command] -h` for more information about command.
```
https://github.com/etcd-io/bbolt/pull/540/files#diff-46e12f11738699c28dd42a6973acce64f7d89f95399d28a611a218dff47f4f9cR59

Reply:
does it make sense to mention it separately as well ? I have no strong opinion. Either way is fine for me